packages/web-components/src/components/va-alert/va-alert.tsx

```diff
@@ -188,13 +186,13 @@ export class VaAlert {
         'usa-alert--success': status === 'continue',
         'usa-alert--slim': slim,
       });
+
       return (
         <Host>
           <div
-            role={this.el.getAttribute('data-role') || role}
```

Comment:
In your screenshot example, you mentioned that it functions with `role="alert"` added to the host element. 

Does that mean that we should let teams know (or updated our guidance) that if they want that functionality, they should do something like: 

```html
<va-alert role="alert/status" ... />
``` 
Is it worthwhile to add a Storybook example?

Reply:
Yeah, good call, I will add an example to Storybook. I'll do a separate PR for docs since it isn't really clear over there as well.

Reply:
@jamigibbs I added those examples and also put up a separate PR for some documentation updates https://github.com/department-of-veterans-affairs/vets-design-system-documentation/pull/3007, thanks!